app/templates/src/main/java/package/security/jwt/_JWTTokenConfigurer.java

```diff
@@ -0,0 +1,25 @@
+package <%=packageName%>.security.xauth;
+
+import org.springframework.security.config.annotation.SecurityConfigurerAdapter;
+import org.springframework.security.config.annotation.web.builders.HttpSecurity;
+import org.springframework.security.core.userdetails.UserDetailsService;
+import org.springframework.security.web.DefaultSecurityFilterChain;
+import org.springframework.security.web.authentication.UsernamePasswordAuthenticationFilter;
+
+public class JWTTokenConfigurer extends SecurityConfigurerAdapter<DefaultSecurityFilterChain, HttpSecurity> {
+
+    private TokenAuthenticationService tokenAuthenticationService;
+
+    private UserDetailsService detailsService;
+
+    public JWTTokenConfigurer(UserDetailsService detailsService, TokenAuthenticationService tokenAuthenticationService) {
```

Comment:
Isn't this a Spring Bean? I need to check but the dependency injection isn't set up